internal/files/files.go

```diff
@@ -0,0 +1,46 @@
+package files
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/elixir-oslo/crypt4gh/keys"
+	"github.com/neicnordic/sda-download/internal/config"
+	log "github.com/sirupsen/logrus"
+)
+
+// GetC4GHKey reads and decrypts and returns the c4gh key
+func GetC4GHKey() (*[32]byte, error) {
```

Comment:
maybe better to return a reader

Reply:
I actually borrowed this from `sda-pipeline` and I think it's working nicely 
https://github.com/neicnordic/sda-pipeline/blob/master/internal/config/config.go#L336